state.go

```diff
@@ -159,12 +159,14 @@ type StateRunner interface {
 	RunState(ctx *StateContext) (State, bool)
 }
 
-// state information that can be used for restring state from storage
+// StateData is state information that can be used for restroing state from storage
```

Comment:
Hehe, that's interesting. I could read the wikipedia text example, could you? :-)